ubcpi/serialize.py

```diff
@@ -242,7 +244,103 @@ def parse_from_xml(root):
         'options': options,
         'rationale_size': {'min': rationale_size_min, 'max': rationale_size_max},
         'correct_answer': correct_answer,
-        'correct_rationale': {'text': correct_rationale},
+        'correct_rationale': correct_rationale,
         'seeds': seeds,
         'algo': {"name": algo, 'num_responses': num_responses}
     }
+
+
+def serialize_options(options, block):
+    """
+    Serialize the options in peer instruction XBlock to xml
+
+    Args:
+        options (lxml.etree.Element): The <options> XML element.
+        block (PeerInstructionXBlock): The XBlock with configuration to serialize.
+
+    Returns:
+        None
+    """
+    for index, option_dict in enumerate(block.options):
+        option = etree.SubElement(options, 'option')
+        # set correct option and rationale
+        if index == block.correct_answer:
+            option.set('correct', u'True')
+
+            if hasattr(block, 'correct_rationale'):
+                rationale = etree.SubElement(option, 'rationale')
+                rationale.text = unicode(block.correct_rationale['text'])
```

Comment:
It seems the texts in block are already encoded. If I add the encode='utf-8', I got the erorr "decoding Unicode is not supported".
Though I added more tests for non-ascii strings as you suggested.